src/components/sort-by.tsx

```diff
@@ -1,23 +1,29 @@
-import { useState } from "react";
+import { useState } from 'react';
 
-import AltArrowDown from "../assets/alt-arrow-down.svg";
-import SortVertical from "../assets/sort-vertical.svg";
-import { cn } from "../lib/utils";
+import { AltArrowDown, SortVertical } from '../assets';
+import { cn } from '../lib/utils';
+import { Order, ProductField, SortOption } from '../types';
 
-export const SortBy = () => {
+interface SortByProps {
+  options: SortOption[];
+  setSort: (sortBy: ProductField, order: Order) => void;
+}
+
+export const SortBy = ({ options, setSort }: SortByProps) => {
```

Comment:
this component definition & name feels off
also setSort could use an object-parameter instead of two positional ones

Reply:
I agree! I'll work on improving the component definition and name, as well as following your recommendation to use an object parameter for setSort. Thanks!